src/model/channel/attachment.rs

```diff
@@ -26,6 +26,8 @@ pub struct Attachment {
     pub url: String,
     /// If the attachment is an image, then the width of the image is provided.
     pub width: Option<u64>,
+    /// The attachment's [media type](https://en.wikipedia.org/wiki/Media_type)
```

Comment:
The link should be put into its own line.
```suggestion
    /// The attachment's [media type].
    ///
    /// [media type]: https://en.wikipedia.org/wiki/Media_type
```